recipes/fasttsne/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "fastTSNE" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c06115ff228bebe1ff8ef1313baa9e6edda81552c96ab9045f1e4d784852d543
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - numpy >1.14
```

Comment:
See:

https://conda-forge.org/docs/meta.html#building-against-numpy

Reply:
Could you please check if this is alright now? I have zero experience packaging conda and I really appreciate your help.